src/Text.js

```diff
@@ -4,8 +4,17 @@ import { fontSize, space, color, responsiveStyle } from 'styled-system'
 
 export const caps = props => props.caps
   ? {
-    textTransform: 'uppercase',
-    letterSpacing: '0.2em'
+    textTransform: 'uppercase'
+  }
+  : null
+
+export const regular = props => props.regular ? { fontWeight: props.theme.regular} : null
+
+export const medium = props => props.medium
+  ? {
+    fontWeight: props.theme.bold,
+    'text-rendering': 'optimizeLegibility',
+    '-webkit-font-smoothing': 'antialiased'
```

Comment:
I think the font-smoothing should only be applied to reversed text (white on dark background) and I think this would be fine to hard-code in the Button component

Reply:
Do you think that needs to be handled differently?  i.e. should `medium` be renamed to `smooth`.

Reply:
I mean, we shouldn't use the font-smoothing prop on normal, dark-on-light applications. The OSes' font rendering engines work fine there. Where people tend to use this property is for white-on-dark applications. When speaking to @odmln we talked about applying this to the Button component specifically, not as a replacement for font weights generally

Reply:
Alright - `medium` is removed, and hardcoded on button instead